spring-ai-chat/accelerator.yaml

```diff
@@ -94,21 +95,6 @@ engine:
             substitutions:
               - text: "<artifactId>ai-chat</artifactId>"
                 with: "'<artifactId>' + #artifactId + '</artifactId>'"
-          - condition: "#vectorStoreType != 'pgvector'"
-            type: ReplaceText
-            regex:
-              pattern: "<dependency>\\s*<groupId>org.springframework.ai</groupId>\\s*<artifactId>spring-ai-pgvector-store</artifactId>\\s*<version>\\$\\{SPRING_AI_VERSION\\}</version>\\s*</dependency>"
```

Comment:
I'm not sure I understand how that regex dealt with the pom, given that the target lines are surrounded with `<!-- -->` but otherwise, LGTM.
Feel free to merge if my comment doesn't alert you.